mystery-deckbuilder/Assets/Scripts/Notepad/Notepad.cs

```diff
@@ -0,0 +1,270 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+public class Notepad : MonoBehaviour
+{
+    //Variables
+    public float numpages;
+    public float numchapters;
+
+    public GameObject notepadCanvas;
+    public GameObject pageImage;
+
+    public bool notebookActive;
+    public int currentChapterID = 0;
+    public int currentPageID = 0;
+
+    public TMP_Text noteText;
+    public TMP_Text chapterTitle;
+    public TMP_Text pageTitle;
+    public Dropdown tabs;
+
+    public List<Chapter> chapterList = new List<Chapter>();
+    public Chapter currentChapter;
+    public Page currentPage;
+
+
+    //Opens or closes the notepad when called
+    public void OpenNotepad()
+    {
+        notebookActive = !notebookActive;
+        notepadCanvas.SetActive(notebookActive);
+    }
+
+    //Changes the current chapter in the notebook
+    public void ChangeTab(int chapID)
+    {
+        //Change current chapterID to the chapterID changed to
+        currentChapterID = chapID;
+        currentChapter = chapterList[currentChapterID];
+        currentPageID=0;
+       //display the new chapter
+        DisplayNotes();
+    }
+
+
+    //Displays all current info to the notebook
+    public void DisplayNotes()
+    {
+        //Display the chapter and page titles
+        chapterTitle.text = currentChapter.GetTitle().ToString();
+        pageTitle.text = currentPage.GetTitle().ToString();
+
+        //Display each note the current page holds
+        foreach(var item in currentPage.notes)
+        {
+            noteText.text = item.ToString();
+        }    
+    }
+
+
+    //Move to the next page in the chapter
+    public void NextPage()
+    {
+        //If next page, exists, go
+        if(currentPageID < (currentChapter.GetNumPages() -1))
+        {
+            currentPageID += 1;
+            DisplayNotes();
+        }  
+    }
+
+    //Move to the previous page in the chapter
+    public void PreviousPage()
+    {
+        //if previous page exists, go
+        if(currentPageID > 0 )
+        {
+            currentPageID -= 1;
+            DisplayNotes();
+        }
+    }
+
+    public void SetDropdown()
+    {
+        //Set tab dropdown menu to contain each of the chapters within chapterList
+        tabs.options.Clear();
+
+        foreach (var item in chapterList)
+        {
+            tabs.options.Add(new Dropdown.OptionData() { text = item.title});
+
+        }
+        DropdownItemSelected(tabs);
+
+        tabs.onValueChanged.AddListener(delegate{ DropdownItemSelected(tabs);});
+
+    }
```

Comment:
For some reason, in testing the notepad the dropdown always defaults to "Option A" at the start. I think this could fixed by modifying the text in the dropdown to be the first chapter, or (more preferred) there may be some way to define the default value!